pynest/nest/lib/hl_api_simulation.py

```diff
@@ -190,99 +190,134 @@ def SetKernelStatus(params):
 
     Other parameters
     ----------------
-    The following parameters can be written to the kernel status dictionary.
+    The following parameters are available from the kernel status dictionary.
+    This is a list of all kernel parameters, the keys flagged with
+    `(read only)` are internally calculated and cannot be user defined.
+    Their values can be consulted using GetKernelStatus.
 
-    resolution : double
+    Time and resolution
+    resolution : float
         The resolution of the simulation (in ms)
-    time : double
-        The current simulation time
-    max_delay : double
+    time : float
+        The current simulation time (in ms)
+    to_do : int, read only
+        The number of steps yet to be simulated
+    max_delay : float
         The maximum delay in the network
-    min_delay : double
+    min_delay : float
         The minimum delay in the network
-    ms_per_tic : double
+    ms_per_tic : float
         The number of milliseconds per tic
-    tics_per_ms : double
+    tics_per_ms : float
         The number of tics per millisecond
     tics_per_step : int
         The number of tics per simulation time step
+    T_max : float, read only
+        The largest representable time value
+    T_min : float, read only
+        The smallest representable time value
+
+    Parallel processing
     total_num_virtual_procs : int
         The total number of virtual processes
-    local_num_threads : int
+    local_num_threads : int, read only
         The local number of threads
+    num_processes : int
+        The number of MPI processes
+    off_grid_spiking : bool, read only
```

Comment:
`off_grid_spiking` is actually writable at present, although it probably should not be. But documentation should document what is, not what should be. Could you correct it here and create a follow-up issue to make `off_grid_spiking` read-only?

Reply:
Done!